cmd/wallet-sdk-gomobile/openid4vp/openid4vp.go

```diff
@@ -44,18 +44,13 @@ type Interaction struct {
 // ActivityLogger is optional, but if provided then activities will be logged there.
 // If not provided, then no activities will be logged.
 type ClientConfig struct {
-	KeyHandleReader         api.KeyReader
-	Crypto                  api.Crypto
-	DIDRes                  api.DIDResolver
-	DocumentLoader          api.LDDocumentLoader
-	ActivityLogger          api.ActivityLogger
-	MetricsLogger           api.MetricsLogger
-	httpClientSkipTLSVerify bool
-}
-
-// DisableHTTPClientTLSVerify disables tls verification, should be used only for test purposes.
-func (c *ClientConfig) DisableHTTPClientTLSVerify() {
-	c.httpClientSkipTLSVerify = true
+	KeyHandleReader api.KeyReader
+	Crypto          api.Crypto
+	DIDRes          api.DIDResolver
+	DocumentLoader  api.LDDocumentLoader
+	ActivityLogger  api.ActivityLogger
+	MetricsLogger   api.MetricsLogger
+	httpClient      *wrapper.HTTPClient
```

Comment:
same here , create http client just in NewInteraction

Reply:
Done